### PR TITLE
std/terminal: lift examples

### PR DESCRIPTION
**Pull request by ringabout**
After this PR:
image

### CODE REVIEW

**Comment by alaviss** [comment]
I'm pretty sure those are unittests and not examples.

**Comment by ringabout** [comment]
What do you mean `unittests`? All working tests should be moved to testament.

**Comment by alaviss** [comment]
They don't look like descriptive examples of how to use the module.

**Comment by ringabout** [comment]
> They don't look like descriptive examples of how to use the module.
Ok. I will work on description later.

File: lib/pure/terminal.nim [hunk 1]
```diff
@@ -905,50 +905,51 @@ proc newTerminal(): owned(PTerminal) =
...
-  stdout.styledWrite(" ordinary text ")
-  stdout.styledWrite(fgGreen, "green text")
-  echo ""
-  echo "ordinary text"
-  stdout.styledWriteLine(fgRed, "red text ", styleBright, "bold red", fgDefault, " bold text")
-  stdout.styledWriteLine(bgYellow, "text in yellow bg", styleBright,
-      " bold text in yellow bg", bgDefault, " bold text")
-  echo "ordinary text"
+runnableExamples:
+  when false:
```
Comment (timotheecour) [inline lib/pure/terminal.nim:909]: * `if false` to make sure this compiles
* maybe even `if defined(nimTerminalEnableTests)`

**Pull request closed without merging**